inventory/openstack/hosts/nova.py

```diff
@@ -34,7 +34,7 @@
 # executed with no parameters, return the list of
 # all groups and hosts
 
-NOVA_CONFIG_FILES = [os.getcwd() + "/nova.ini",
+NOVA_CONFIG_FILES = [os.path.join(os.path.dirname(os.path.realpath(__file__)), "nova.ini"),
```

Comment:
I agree with you that this change is not related at all to GCE ... however this change makes sense : 

nova.ini is located in the same directory than nova.py
Getcwd will return the location from where we execute the script nova.py resulting in an error if the script is executed from directory different. So this is fixing that issue.

On AWS, GCE, LIBVIRT we take the .ini file in the same location as the .py
So I think it was just a mistake to use getcwd for Openstack

Anyways we can take this change off and make an other PR.
It's up to you to judge if it's worth for a minor change.